be/src/exec/vectorized/hdfs_scanner.h

```diff
@@ -185,11 +202,16 @@ class HdfsScanner {
     virtual Status do_get_next(RuntimeState* runtime_state, ChunkPtr* chunk) = 0;
     virtual Status do_init(RuntimeState* runtime_state, const HdfsScannerParams& scanner_params) = 0;
 
+    void in_pending_queue();
```

Comment:
nit: maybe enter_pending_queue and exit_pending_queue can be easier to understand?

Reply:
Good idea. Will fix that.